src/components/EditorContainer.jsx

```diff
@@ -28,6 +28,12 @@ function EditorContainer(props) {
       >
         {t(`languages.${props.language}`)}
       </div>
+      <div
+        className="environment__label_toggleTextSize label"
+        onClick={props.onToggleEditorTextSize}
+      >
+        Aa
```

Comment:
It’d probably be slick to use [fa-search-plus](http://fontawesome.io/icon/search-plus/) and [fa-search-minus](http://fontawesome.io/icon/search-minus/) respectively (we already have FontAwesome installed)